docs/introduction_tutorial/intro_tutorial_part3.ipynb

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
<p>I think we need a lot more detail about what to put in the meta.yaml file. This document should be a useful reference: https://github.com/pangeo-forge/roadmap/blob/master/doc/adr/0002-use-meta-yaml-to-track-feedstock-metadata.md</p><p> </p><p>We should show a completed meta.yaml for this recipe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>